.circleci/config.yml

```diff
@@ -102,6 +102,8 @@ jobs:
     <<: *container_config_node
     steps:
       - *attach_workspace
+      - node/install-npm:
+          version: "8.5.3"
```

Comment:
**question**: is npm used in this job at all? why is this needed?

Reply:
gotcha. does it need to be that specific version?

Reply:
yes, because if it's version: "8" then it installs 8.6.0 and o-toggle somehow doesn't get bundled with this version (it also doesn't get bundled with npm 8.6.0 and node 14.19.1, but works fine locally with npm 8.5.3 and node 14.19.1 🤯 )

Reply:
huh! seems weird that having a different version for the build command would cause that? happy for you to merge  but it would be good to get to the bottom of this